test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/ObjectManagement/DatabaseHandlerTests.cs

```diff
@@ -162,6 +166,111 @@ public async Task DatabaseAlreadyExistsErrorTest()
             }
         }
 
+        [Test]
```

Comment:
It'd be great to see some tests for the helper method functionality (such as GetAzureServiceLevels) too

Reply:
We already do that indirectly, since the AzureServiceLevels property is a statically cached result of that method.